web/app/themes/ppj/partials/videoPlayer.php

```diff
@@ -25,13 +25,17 @@ class="video-player__cover"
         >
             <?php echo $coverImage ?>
             <div class="video-player__play-button-container">
-                <svg class="video-player__play-button"
+                <svg
+                    aria-labelledby="playButtonIconTitle playButtonIconDesc"
+                    class="video-player__play-button"
                      x="0px"
                      y="0px"
                      viewBox="0 0 138.3 138.3"
                      style="enable-background:new 0 0 138.3 138.3;"
                      role="img"
                 >
+                    <title id="playButtonIconTitle">PlayButton icon</title>
+                    <desc id="playButtonIconDesc">An white triangle pointing to the right, within a circle</desc>
```

Comment:
- Missing a space between "Play" and "Button" in the `<title>` tag.
- Should be "a" white triangle, not "an" in the `<desc>` tag.

Reply:
done